test/spspSpec.js

```diff
@@ -118,11 +118,11 @@ describe('SPSP Module', function () {
         .get('/spsp')
         .reply(200, spspResponse)
 
-      const payment = yield SPSP.quoteDestination(this.plugin, 'alice@example.com', '10')
+      const payment = yield SPSP.quoteSource(this.plugin, 'alice@example.com', '12')
       assert.deepEqual(payment, {
         destinationAccount: "example.alice",
         connectorAccount: "example.connie",
-        sourceAmount: "10",
+        sourceAmount: "12",
```

Comment:
the mocks return a default value for the unspecified one of `sourceAmount` or `destinationAmount` to simulate a quote on ILP core, so I'd need a way to pass the unknown amount through the mocks